exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricConfiguration.java

```diff
@@ -96,16 +120,24 @@ public abstract static class Builder {
 
     abstract Duration getDeadline();
 
+    /** Set the GCP project where metrics should be writtten. */
```

Comment:
```suggestion
    /** Set the GCP project where metrics should be writtten. Defaults to the application default credential's project.*/
```
or something like that.